sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ChangeFeedProcessorOptions.java

```diff
@@ -9,10 +9,10 @@
  * Specifies the options associated with {@link ChangeFeedProcessor}.
  */
 public class ChangeFeedProcessorOptions {
-    private static Duration DefaultRenewInterval =  Duration.ofMillis(0).plusSeconds(17);
-    private static Duration DefaultAcquireInterval = Duration.ofMillis(0).plusSeconds(13);
-    private static Duration DefaultExpirationInterval = Duration.ofMillis(0).plusSeconds(60);
-    private static Duration DefaultFeedPollDelay = Duration.ofMillis(0).plusSeconds(5);
+    private static final Duration DEFAULT_RENEW_INTERVAL = Duration.ofMillis(0).plusSeconds(17);
+    private static final Duration DEFAULT_ACQUIRE_INTERVAL = Duration.ofMillis(0).plusSeconds(13);
+    private static final Duration DEFAULT_EXPIRATION_INTERVAL = Duration.ofMillis(0).plusSeconds(60);
+    private static final Duration DEFAULT_FEED_POLL_DELAY = Duration.ofMillis(0).plusSeconds(5);
```

Comment:
I understand this is not part of your PR change, but it will be great if we could have renamed them in a way that explains their unit as well, like `DEFAULT_RENEW_INTERVAL_IN_SECONDS`

Reply:
Not related to checkstyle, so not including in this PR